source/features/show-forks-in-search.tsx

```diff
@@ -0,0 +1,33 @@
+import './show-forks-in-search.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import SearchQuery from '../github-helpers/search-query';
+
+function init(): void {
+	const includeForks = new SearchQuery(location.search).includes('fork:true');
+	const checkbox = <label className="float-right"><input className="rgh-show-forks-in-search-checkbox mr-1" type="checkbox" checked={includeForks}/>Include forks</label>;
+	checkbox.addEventListener('click', () => {
```

Comment:
If the page has PJAX enabled, this listener will be lost on `history.back()`. Try and if it breaks, it needs to use `delegate`

Also it should probably be `change` instead of `click`

Reply:
> Also it should probably be `change` instead of `click`

I noticed this, but haven't fixed it yet. Well it says `checkbox` but it is a in fact a label 😅